sites/next.skeleton.dev/src/components/docs/FrameworkPicker.astro

```diff
@@ -1,18 +1,6 @@
 ---
 import { Icon } from 'astro-icon/components';
-
-export const frameworks = [
-	{
-		name: 'React',
-		slug: 'react',
-		iconName: 'react',
-	},
-	{
-		name: 'Svelte',
-		slug: 'svelte',
-		iconName: 'svelte',
-	},
-];
+import { frameworks } from 'src/lib/utils';
```

Comment:
See my comment regarding the path for this script in the Conversation tab. Just calling it here so this is not missed.